devel/bun/bun/Portfile

```
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem  1.0
+PortGroup npm 1.0
+
+name                bun
+version             1.1.10
+maintainers         johnlindop.com:git
```

Please add your GitHub handle:

```suggestion
maintainers         {johnlindop.com:git @JLindop}
```

Are you ok with other people making minor changes to this port without asking for your permission? If so, please add `openmaintainer` as well:

```suggestion
maintainers         {johnlindop.com:git @JLindop} openmaintainer
```